spec/PushController.spec.js

```diff
@@ -495,7 +495,7 @@ describe('PushController', () => {
       });
   });
 
-  it('properly creates _PushStatus', done => {
+  xit('properly creates _PushStatus', async () => {
```

Comment:
As much as I would *love* to see this merged, I don't feel comfortable given that there are so many xits for essential features.